src/cli/commands/dev.ts

```diff
@@ -71,15 +71,17 @@ export class Dev implements Command {
     }
 
     const startModule = createStartModuleContent({
-      registerTypeScript: layout.tsConfig.content.options,
+      registerTypeScript: {
+        ...layout.tsConfig.content.options,
+        module: ts.ModuleKind.CommonJS,
+        target: ts.ScriptTarget.ES2015,
```

Comment:
@Weakky about your comment here https://github.com/graphql-nexus/nexus/pull/743#discussion_r415962381.

I don't have a perfect answer. What I do know is that we need to return something runnable by Node, though.

I think we need some docs about this, for one thing.

If users want to go nuts, start adding e.g. experimental flags or use node 15... well I would say we don't support that right now?